File: VB/App_Code/Source.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web

''' <summary>
''' Summary description for Source
''' </summary>
Public Class Source
    Private Const count As Integer = 5
    Public Function PopulateList() As List(Of Category)
        Dim list As New List(Of Category)()
        For i As Integer = 0 To 4
            list.Add(New Category() With {.CategoryID = i, .CategoryName = "Name" & i, .Description = "Description" & i})
        Next i
        Return list
    End Function
End Class

File: CS/App_Code/Source.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

/// <summary>
/// Summary description for Source
/// </summary>
public class Source {
    const int count = 5;
    public List<Category> PopulateList() {
        List<Category> list = new List<Category>();
        for(int i = 0; i < 5; i++) {
            list.Add(new Category() { CategoryID = i, CategoryName = "Name" + i, Description = "Description" + i });
        }
        return list;
    }
}